apps/zupoll-client/src/api/prismaTypes.ts

```diff
@@ -21,6 +21,8 @@ export type Ballot = {
   voterSemaphoreGroupRoots: string[];
   ballotType: BallotType;
   expiryNotif: ExpiryNotifStatus | null;
+  pipelineId?: string;
+  isPublic: boolean;
```

Comment:
todo: https://linear.app/0xparc-pcd/issue/0XP-599/allow-for-publicly-viewable-ballots-voting-still-restricted-to-group